listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -89,7 +89,34 @@ def get_listens_for_training_model_window(metadata):
     except FileNotFetchedException as err:
         current_app.logger.error(str(err), exc_info=True)
         sys.exit(-1)
-    return training_df
+    return utils.get_listens_without_artist_and_recording_mbids(training_df)
+
+def get_mapped_artist_and_recording_msids(partial_listens_df, artist_mapping_df, recording_mapping_df):
```

Comment:
This fetches MBIDs, no? Would a better name for this function be get_mapped_artist_and_recording_mbids ?

Reply:
definitely :)